tests/sav/hello_ios.res

```diff
@@ -0,0 +1 @@
+hello_ios.bin is a directory
```

Comment:
No. At this point we don't have anything to test the execution of graphical applications, even less on iOS.